.prettierignore-sync

```diff
@@ -7,6 +7,7 @@ packages/{*,labs/*}/{.gitignore,.prettierignore}
 packages/{lit-starter-ts,lit-starter-js}/.eslintignore
 
 [inline]
+package-lock.json
```

Comment:
This is going to result in some really big diffs of all the package-lock.json files as we go from the prettier diff versions to the formatting that npm writes

Reply:
One time yeah. (Also, you read package-lock diffs?)

Reply:
sometimes!